cmd/minikube/cmd/update-context.go

```diff
@@ -49,9 +49,9 @@ var updateContextCmd = &cobra.Command{
 			exit.WithError("update config", err)
 		}
 		if updated {
-			console.OutStyle("celebrate", "IP has been updated to point at %s", ip)
+			console.OutStyle("celebrate", "IP has been updated to point at %s", machineName)
```

Comment:
This is still slightly confusing for me. How about:

"%s IP has been updated to point to %s, machineName, ip

Reply:
@tstromberg done